src/Test/DatabaseResetter.php

```diff
@@ -93,4 +89,30 @@ private static function createApplication(KernelInterface $kernel): Application
 
         return $application;
     }
+
+    private static function createORMDatabaseResetter(Application $application, KernelInterface $kernel): ORMDatabaseResetter
+    {
+        $container = $kernel->getContainer();
+        $configuration = $container->get(Configuration::class);
```

Comment:
@kbond this breaks the tests which are not using the bundle. 

Should we maintain "bundleless" usage until 2.0 or do you think this could be acceptable?